cmake/env/pol_global_config.h.in

```diff
@@ -9,11 +9,10 @@
 #cmakedefine HAVE_MYSQL
 #cmakedefine ENABLE_BENCHMARK
 
-#define POL_VERSION @POL_VERSION@
 #define POL_VERSION_STR "@POL_VERSION_STR@"
 #define POL_VERSION_NAME "@POL_VERSION_NAME@"
-#define POL_VERSION_ID "POL@POL_VERSION_STR@ @POL_VERSION_NAME@"
+#define POL_VERSION_ID "@POL_VERSION_STR@ @POL_VERSION_NAME@"
 #define POL_BUILD_ARCH_BITS_STR "@ARCH_BITS@"
-#define POL_COPYRIGHT "Copyright (C) 1993-2018 Eric N. Swanson"
+#define POL_COPYRIGHT "Copyright (C) 1993-2020 Eric N. Swanson"
```

Comment:
Should we use the chance to move this into CMakeLists.txt? Then we have copyright and version in a single spot.

Reply:
Sounds good to me.  Want to push that change to this branch?